x-pack/test/api_integration/apis/telemetry/telemetry.ts

```diff
@@ -42,6 +42,19 @@ function getCacheDetails(body: UnencryptedTelemetryPayload): CacheDetails[] {
   return body.map(({ stats }) => (stats as UsageStatsPayload).cacheDetails);
 }
 
+function updateClusterUuidInLogstashStats(
```

Comment:
review note: monitoring cluster UUID always changes when `functional_test_server` runs. This change updates LS stats monitoring cluster UUID with a static UUID to keep data identical with fixtures.

Locally validated with test run:
```
node scripts/functional_test_server --config=x-pack/test/api_integration/apis/telemetry/config.ts
node scripts/functional_test_runner --config=x-pack/test/api_integration/apis/telemetry/config.ts
```